baseplate/clients/redis_cluster.py

```diff
@@ -498,4 +498,37 @@ def execute_command(self, *args: Any, **kwargs: Any) -> Any:
     # pylint: disable=arguments-differ
     def execute(self, **kwargs: Any) -> Any:
         with self.server_span.make_child(self.trace_name):
-            return super().execute(**kwargs)
+
+            success = "true"
+            start_time = perf_counter()
+            labels = {
+                f"{PROM_LABELS_PREFIX}_command": "pipeline",
+                f"{PROM_LABELS_PREFIX}_client_name": self.connection_pool.connection_kwargs.get(
+                    "client_name", ""
+                ),
+                f"{PROM_LABELS_PREFIX}_database": self.connection_pool.connection_kwargs.get(
+                    "db", ""
+                ),
+                f"{PROM_LABELS_PREFIX}_type": "standalone",
+            }
+            num_reqs = len(self.command_stack)
+            commands = [cmd.args[0] for cmd in self.command_stack]
+
+            try:
+                with ACTIVE_REQUESTS.labels(**labels).track_inprogress():
```

Comment:
[under the hood](https://sourcegraph.com/github.com/prometheus/client_python/-/blob/prometheus_client/context_managers.py?L37:7), this still uses `inc` and `dec`, can you please add a test for this (happy path and a case where `execute` returns an exception)

Reply:
@Xaelias Any clue as how to do this? Looking at https://github.com/reddit/baseplate.py/pull/731/files#diff-a91d47420f289905a951c1be7b82eb613e3d3ca01e72871564b122a6a715774a the tests were just removed.

Reply:
I still don't see a test for this, I think I'd be satisfied by setup resetting the counter for calls and just checking that inc and dec were both called once and the resulting value is 0 (which you already have)

Reply:
The old strategy to test doesn't seem to work with this. @Xaelias Did you spend any effort figuring out how to test this for #731? 

Happy to revert to using `inc` and `dec` as this PR originally had.